sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -511,7 +511,7 @@ namespace Azure { namespace Core { namespace Http {
     /**
      * @brief Get HTTP response body as #Azure::Core::IO::BodyStream.
      */
-    std::unique_ptr<Azure::Core::IO::BodyStream> GetBodyStream()
+    std::unique_ptr<Azure::Core::IO::BodyStream> ExtractBodyStream()
```

Comment:
Does this adhere to the most recent apiview discussion on `RawResponse`?
https://apiview.dev/Assemblies/Review/9975bc9d035a4c059ff3cf0c1dadf087#Azure::Core::Http::RawResponse

Reply:
Yesterday we specifically highlighted that there is a PR in review that would update the naming to `Extract()`